spec/controllers/v2/editions_controller_spec.rb

```diff
@@ -0,0 +1,146 @@
+require "rails_helper"
+
+RSpec.describe V2::EditionsController do
+  def u(string)
+    CGI.escape(string)
+  end
+
+  describe "index" do
+    before do
+      Timecop.freeze("2017-01-01 09:00:00") do
+        50.times do |index|
+          FactoryGirl.create(
+            :draft_edition,
+            id: index + 1,
+            document: FactoryGirl.create(:document, locale: "fr"),
+            base_path: "/content#{index + 1}",
+          )
+        end
+
+        50.times do |index|
+          FactoryGirl.create(
+            :live_edition,
+            id: index + 51,
+            document: FactoryGirl.create(:document),
+            base_path: "/content#{index + 51}",
+          )
+        end
+
+        50.times do |index|
+          FactoryGirl.create(
+            :unpublished_edition,
+            id: index + 101,
+            document: FactoryGirl.create(:document),
+            base_path: "/content#{index + 101}",
+            publishing_app: "test",
+          )
+        end
+      end
```

Comment:
This looks like a lot of DB IO to run before each test. Can we maybe reduce it so we only need to do this whole lot for something testing pagination and then just add a single item for when we're checking other aspects?